### PR TITLE
<CustomModalLayout/> and <BaseModalLayout/> - add legacy drivers and fix styles

### PR DESCRIPTION
**Pull request by bamai**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: src/BaseModalLayout/BaseModalLayout.legacy.driver.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import buttonDriverFactory from '../Button/Button.legacy.driver';
```
Comment (shlomitc) [inline src/BaseModalLayout/BaseModalLayout.legacy.driver.js:1]: Make sure the that BaseModalLayout tests run aginst this file as well (look at `Checkbox.spec.js` for example)

File: src/CustomModalLayout/CustomModalLayout.legacy.driver.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { baseModalLayoutDriverFactory } from '../BaseModalLayout/BaseModalLayout.legacy.driver';
```
Comment (shlomitc) [inline src/CustomModalLayout/CustomModalLayout.legacy.driver.js:1]: same comment regarding running the tests

File: src/CustomModalLayout/CustomModalLayout.st.css [hunk 3]
```diff
@@ -50,3 +50,11 @@
 .root::footnote::innerContent {
   justify-content: flex-start;
 }
+
+.root::content::innerContent {
```
Comment (shlomitc) [inline src/CustomModalLayout/CustomModalLayout.st.css:54]: is this needed?

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5907/

File: src/CustomModalLayout/CustomModalLayout.st.css (hunk 3 above)
Comment (bamai) [reply src/CustomModalLayout/CustomModalLayout.st.css:54]: yes, otherwise the content doesn't get full height

**Review by shlomitc — APPROVED** [review]

**Pull request merged**